README.md

```diff
@@ -79,11 +83,24 @@ See [Install Docker Compose](https://docs.docker.com/compose/install/) for detai
    chown -R root docker/etc/cron.d
    ```
 
-8. Launch the containers. It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
+8. Launch the containers.
+
+- With Docker Compose:
+
+   It is recommeded to wait 1 minute in order for each container to configure fully before launching the next
```

Comment:
Is there a way to check the status of a container before starting the next?

Reply:
Yes and no, mostly no. 

In theory there are ways to do it (like the `ansible` [wait_for](http://docs.ansible.com/ansible/latest/wait_for_module.html) module or `docker-compose`'s [healthcheck](https://docs.docker.com/compose/compose-file/#healthcheck)), but the main problem is the standard image database container reports it's ready when the port is opened which isn't the same as ready to accept connections, which tends to scupper those options

I can get rid of the wait between the Server and the REST interface as they are independent.